src/server/rollbar.js

```diff
@@ -12,6 +12,7 @@ var urllib = require('url');
 var jsonBackup = require('json-stringify-safe');
 
 var Telemeter = require('../telemetry');
+var Instrumenter = require('./telemetry');
```

Comment:
Nit: this _looks_ like a typo, since the line immediately before references the parent directory while this references the current directory. I'd suggest - in a separate PR - renaming the files to better match their intent; perhaps:

* `../telemetry` => `../telemeter`
* `./telemetry` => `./instrumenter`

Reply:
I tend to agree. For this PR, I wanted to stick with the current naming convention. Changing it, while keeping the code base as a whole consistent, would mean touching a number of files outside the scope of this PR.